src/types/positions.ts

```diff
@@ -186,6 +191,26 @@ export interface BaseToken extends AbstractToken {
   type: 'base-token'
 }
 
+export interface RawTokenInfo {
+  address?: string
+  name: string
+  symbol: string
+  decimals: number
+  imageUrl: string
+  tokenId: string
+  networkId: NetworkId
+  isNative?: boolean
+  priceUsd?: string
+}
```

Comment:
nit: could we move this one to be internal only to `getPositions.ts`? I didn't want to expose it yet.